lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -90,7 +90,9 @@
                             model: userAccountModel,
                             title: gettext('Email Address'),
                             valueAttribute: 'email',
-                            helpMessage: gettext('The email address you use to sign in to this site. Communications from us and your courses are sent to this address.')
+                            helpMessage: interpolate_text(
+                                gettext('The email address you use to sign in. Communications from {platform_name} and your courses are sent to this address.'), {platform_name: platformName}
```

Comment:
Not a fork, see above. Our previous version was our way of removing edX mention, then upstream updated it the string themselves, so we are taking theirs now.